### PR TITLE
Wb/docker change

### PR DESCRIPTION
**Pull request by SarthakSingh31**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,24 +2,31 @@ version: '3.4'
...
     restart: always
     container_name: app_name_devtools
-    image: node:14.5.0-stretch
     volumes:
-      - ./package.json:/devtools/axdd-django-vue/package.json
-      - ./webpack.config.js:/devtools/axdd-django-vue/webpack.config.js
-      - ./app_name:/devtools/axdd-django-vue/app_name
-    working_dir: /devtools/axdd-django-vue
-    command: bash -c "yes yes | npm install; npx webpack --watch --mode=development"
+      - ./myuw:/app/myuw
```
Comment (abztrakt) [inline docker-compose.yml]: What should this be? If this is a generic "app_name" template, should that be ./app_name:/app/app_name instead?

**Review by charlon — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -11,29 +11,28 @@ ADD --chown=acait:acait requirements.txt /app/
...
-
-FROM node:14.5.0-stretch AS wpack
-ADD . /app/
+ADD ./package.json /app/
 WORKDIR /app/
 RUN npm install .
+
+ADD . /app/
+ARG VUE_DEVTOOLS
+ENV VUE_DEVTOOLS=$VUE_DEVTOOLS
```
Comment (abztrakt) [inline Dockerfile:27]: Is this actually used by anything here?

File: docker-compose.yml [hunk 3]
```diff
@@ -2,24 +2,31 @@ version: '3.4'
...
     container_name: app_name_devtools
-    image: node:14.5.0-stretch
     volumes:
-      - ./package.json:/devtools/axdd-django-vue/package.json
-      - ./webpack.config.js:/devtools/axdd-django-vue/webpack.config.js
-      - ./app_name:/devtools/axdd-django-vue/app_name
-    working_dir: /devtools/axdd-django-vue
-    command: bash -c "yes yes | npm install; npx webpack --watch --mode=development"
+      - ./app_name:/app/app_name
+      - statics-volume:/static
```
Comment (abztrakt) [inline docker-compose.yml:27]: Why two volumes?

File: docker-compose.yml [hunk 4]
```diff
@@ -2,24 +2,31 @@ version: '3.4'
...
+    command: npx webpack --watch --mode=development
+    stdin_open: true
+    tty: true
+volumes:
+  statics-volume:
```
Comment (abztrakt) [inline docker-compose.yml:32]: ?

File: docker/settings.py [hunk 5]
```diff
@@ -3,16 +3,13 @@
...
 ]
 
-WEBPACK_LOADER = {
-    'DEFAULT': {
-        'BUNDLE_DIR_NAME': 'app_name/bundles/',
-        'STATS_FILE': os.path.join(BASE_DIR, 'app_name', 'static', 'webpack-stats.json'),
-    }
-}
+STATICFILES_DIRS = [
+    '../static/app_name/',
```
Comment (abztrakt) [inline docker/settings.py]: Why were we doing this relatively?

**Review by abztrakt — CHANGES_REQUESTED** [review]

File: docker/settings.py (hunk 5 above)
Comment (charlon) [reply docker/settings.py]: should be '/static/app_name/'

File: webpack.config.js [hunk 6]
```diff
@@ -17,32 +17,23 @@ module.exports = {
...
-                }
-            }
-        }
+          chunks: 'all',
+        },
     },
 
     output: {
-        path: path.resolve('./app_name/static/app_name/bundles/'),
+        path: path.resolve('../static/app_name/'),
```
Comment (charlon) [inline webpack.config.js]: make this absolute!

File: package.json [hunk 7]
```diff
@@ -43,8 +44,7 @@
     "vue-router": "^3.1.6",
     "vue-template-compiler": "^2.6.11",
     "vue-underscore": "^0.1.4",
-    "webpack": "^4.42.1",
-    "webpack-bundle-tracker": "^0.4.3",
+    "webpack": "^4.46.0",
```
Comment (charlon) [inline package.json]: update to webpack 5

File: app_name/templates/index.html [hunk 8]
```diff
@@ -17,17 +17,14 @@
     <!-- base js libs -->
     <script src="https://code.jquery.com/jquery-3.3.1.js" integrity="sha256-2Kok7MbOyxpgUVvAk/HJ2jigOSYS2auK4Pfzbm7uH60=" crossorigin="anonymous"></script>
```
Comment (charlon) [inline app_name/templates/index.html]: remove jquery!

**Review by abztrakt — APPROVED** [review]

**Pull request merged**